install-and-configure/install/multi-cluster/federated-etl/thanos-migration-guide.md

```diff
@@ -0,0 +1,154 @@
+# Migration Guide from Thanos to Kubecost 2.0 (Aggregator)
+
+This tutorial is intended to help our users migrate from the legacy Thanos federation architecture to [Kubecost 2.0 (Aggregator)](aggregator.md). There are a few requirements in order to successfully migrate to Kubecost 2.0. This new version of Kubecost includes a new backend Aggregator which handles the ETL data built from source metrics more efficiently. Kubecost 2.0 will provide new features, optimize the Kubecost's UI performance, and enhance the user experience.
+
+## Key changes
+* Assets and Allocations are now paginated
+   * Frontend/API users never have full set of information
```

Comment:
Can someone explain this a little better? FE + API users encompasses all Kubecost users right?

Reply:
so, I guess there is an important distinction here. The FE users don't have the full set of information, because the FE uses pagination. That's why we now have a separate 'topline' call that gives them the totals for the top line display. 

However - if you leave off the pagination parameters (limit= and offset=), then the endpoints default to returning _everything_ . So I guess an API user that did not include limit/offset would get everything back, the full information set

Reply:
Didn't we make limit/offset mandatory for certain APIs? Also, haven't FE users always had pagination? We just recently changed the name of the parameter for it

Reply:
I would remove this bullet entirely. "Frontend/API users never have full set of information" isn't a correct statement; it is enough to say that there is now pagination and (below) offset/limit govern this.